src/Tribe/Updater.php

```diff
@@ -198,10 +185,15 @@ public function flush_rewrites() {
 		add_action( 'wp_loaded', 'flush_rewrite_rules' );
 	}
 
+	/**
+	 * Set the Capabilities for Events and Related Post Types.
+	 *
+	 * @since TBD - change method of calling set_capabilities.
+	 */
 	public function set_capabilities() {
-		$this->capabilities = new Tribe__Events__Capabilities();
-		add_action( 'wp_loaded', array( $this->capabilities, 'set_initial_caps' ) );
-		add_action( 'wp_loaded', array( $this, 'reload_current_user' ), 11, 0 );
+
+		tribe('tec.capabilities')->init_set_caps();
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: cff07191e20fb1dc111971e935a3bf36</sub>